utils/config_file_impl.hh

```diff
@@ -22,6 +22,15 @@
 
 #include <seastar/json/json_elements.hh>
 
+namespace utils {
+
+template <typename T>
+T config_from_string(std::string_view string_representation) {
+    return config_from_string<T>(string_representation);
```

Comment:
This looks like a typo, and infinite loop - didn't you intend to have boost::lexical_cast here?
Or am I misunderstanding what I'm seeing?

Reply:
how did it work? it's an infinite loop.

Reply:
Maybe you only tested the bool specialization :-)

Reply:
Indeed other tests fail.